classyScheduleJavafx/.idea/runConfigurations/Main.xml

```diff
@@ -1,16 +0,0 @@
-<component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="Main" type="Application" factoryName="Application" nameIsGenerated="true">
-    <option name="MAIN_CLASS_NAME" value="sample.Main" />
-    <module name="ClassySchedule" />
-    <option name="VM_PARAMETERS" value="--module-path ${PATH_TO_FX};out/production --add-modules javafx.controls,javafx.fxml,javafx.base,javafx.graphics" />
-    <extension name="coverage">
-      <pattern>
-        <option name="PATTERN" value="sample.*" />
-        <option name="ENABLED" value="true" />
-      </pattern>
-    </extension>
-    <method v="2">
-      <option name="Make" enabled="true" />
-    </method>
-  </configuration>
-</component>
```

Comment:
This was an old version of Desktop Teams' _classySchedueJavaFx_ folder. We deleted it so that when we merge the Database-Branch into Main after our demo, we will not be pushing their old code on top of any changes they have made.